### PR TITLE
Extended Support to Enclosure EG, Scope, UplinkSet, and Network Resources

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
Extended Support to Enclosure EG, Scope, UplinkSet, and Network Resources

### Issues Resolved
N/A

### Check List
- [ ] New functionality includes testing.
  - [ ]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
+- This release supports API1800 minimally where we can use OneView API1800 with this SDK. No new fields are added/deleted to support API1800.
+
+### Oneview Features supported
+- Enclosure
+- Enclosure Group
+- Ethernet Network
+- FC Network
+- FCOE Network
+- Uplink Set
+- Scopes
```
Comment (yuvirani) [inline CHANGELOG.md:15]: Correct the order

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,19 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+# [v1.4.0]
```
Comment (yuvirani) [inline CHANGELOG.md:4]: it should be v1.4.0 (unrelased)

Comment (AsisBagga) [reply CHANGELOG.md]: @yuvirani done

File: CHANGELOG.md (hunk 1 above)
Comment (AsisBagga) [reply CHANGELOG.md:14]: @yuvirani done

**Pull request merged**